src/nile/signer.py

```diff
@@ -68,6 +83,18 @@ def from_call_to_call_array(calls):
     return (call_array, calldata)
 
 
+def get_declare_hash(sender, contract_class, max_fee, nonce):
+    """Compute the hash of a declare transaction."""
+    return calculate_declare_transaction_hash(
+        contract_class=contract_class,
+        chain_id=StarknetChainId.TESTNET.value,
```

Comment:
At some point, we're going to need a param to adjust the chain id based on the network (for `get_transaction_hash` as well). I just created an issue #234.